core/models/option.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+from . import Program, Form, Question
+
+class Option(models.Model):
+    program = models.ForeignKey(Program, null=True, on_delete=models.CASCADE)
```

Comment:
having program is somewhat redundant here, will confirm

Reply:
Yeah, I did see that all the information was on every table. But rather than assuming we would be scanning the table for the key, I included the foreign keys on the table like in the google doc.

Reply:
We can remove program id from `option` and `question`. having program id on `form` is sufficient, and then , we filter by form id. i see this working as you query form table to get a list of forms for that program, and then use the selected form's id to go get the actual form questions.